JSDemos/Demos/TabPanel/SortableClosableTabs/jQuery/data.js

```diff
@@ -273,7 +273,7 @@ const tasks = [{
   StartDate: '2013/03/16',
   DueDate: '2013/03/26',
   Status: 'Need Assistance',
-  Priority: 'Normal',
+  priority: 'medium',
```

Comment:
Other values in priority property starts with uppercase, 'High', 'Urgent'

Reply:
This is a mistake in another demo, fixed here [e3146bb4c11cd73ee303de1c3dacc5f6f92e1eb5].